.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Development Build - Linux
+env:
+ BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, reopened, edited]
+  push:
+    paths-ignore:
+      - "README.md"
+      - ".gitignore"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
probably best to pin the ubuntu version to for example `ubuntu-20.04` and then use dependabot to update the workflows, see https://github.com/maplibre/maplibre-gl-js/blob/11a1262601d9842efb4f6a44fed40fcb55fc2462/.github/dependabot.yml#L14 for an example